frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -42,28 +22,29 @@ def __init__(self, request_model, *a, **kw):
         self.send_body = True if 'send_body' in kw and kw['send_body'] else False
 
     def encode_add_seeds(self, seeds):
-        return packb([b'as', [_prepare_request_message(seed) for seed in seeds]])
+        return packb([b'as', [_prepare_request_message(seed) for seed in seeds]], use_bin_type=True, encoding="utf-8")
```

Comment:
according to short msgpack-python documentation, it's recommended to use encoding argument for ``unpackb`` not for ``packb``